config_examples/README.md

```diff
@@ -0,0 +1,45 @@
+# Collector example configurations
+
+Here you can find a collection of example Collector configurations.
+
+## Samples
+
+- [Drop](drop.yaml)
+- [Effective](effective.yaml.yaml)
+- [Health check](healthcheck.yaml)
+- [Jaeger Receiver](jaeger.yaml)
+- [Pipeline](pipeline.yaml)
+- [Tail sampling](tail_sampling.yaml)
+
+## Sending data to Dynatrace
+
+In addition to the `debug` exporter, some samples are also configured with the `otlphttp` exporter
+so you can also see the data in your Dynatrace environment.
+
+Before running the samples, make sure you have the following environment variables set:
+
+- `DT_OTLP_ENDPOINT`: The OTLP HTTP endpoint of your Dynatrace environment.
+  - Follow the guide: [Export to ActiveGate](https://docs.dynatrace.com/docs/shortlink/otel-getstarted-otlpexport#export-to-activegate)
+    to see how to get the correct API URL for your environment
+- `API_TOKEN`: The Dynatrace API access token. Follow the guide on [Authentication](https://docs.dynatrace.com/docs/shortlink/otel-getstarted-otlpexport#authentication-export-to-activegate) to see the scopes required for ingesting OTLP data.
+
+## Trying out the configuration examples
```

Comment:
Does setting the environment variables in the user's shell pass them through to the Docker container, or do we need to pass `--env` flags?

Reply:
Yep, forgot about it. I fixed it now and copied/paste the example and traces show up as expected.